components/firebase_admin_sdk/actions/create-realtime-db-record/create-realtime-db-record.mjs

```diff
@@ -35,4 +35,14 @@ export default {
       await new Promise((resolve) => setTimeout(resolve, 5000));
     },
   },
+  async run({ $ }) {
+    try {
+      await this.firebase.initializeApp();
+      const response = await this.getResponse();
+      this.emitSummary($);
+      return response;
+    } finally {
+      await this.deleteApp();
```

Comment:
Maybe this is supposed to be `this.firebase.deleteApp` since we deleted the `deleteApp` from `app`?

Reply:
Oh, ok, my bad